bigbluebutton-html5/imports/ui/components/audio/audio-modal/container.jsx

```diff
@@ -21,7 +21,10 @@ const isRTL = document.documentElement.getAttribute('dir') === 'rtl';
 export default lockContextContainer(withModalMounter(withTracker(({ mountModal, userLocks }) => {
   const listenOnlyMode = getFromUserSettings('bbb_listen_only_mode', APP_CONFIG.listenOnlyMode);
   const forceListenOnly = getFromUserSettings('bbb_force_listen_only', APP_CONFIG.forceListenOnly);
-  const skipCheck = getFromUserSettings('bbb_skip_check_audio', APP_CONFIG.skipCheck);
+  const AUDIO_TEST_NUM_KEY = 'EchoTestNumber';
+  let checknumber = sessionStorage.getItem(AUDIO_TEST_NUM_KEY);
+  const skipCheck = !checknumber  ? getFromUserSettings('bbb_skip_check_audio', APP_CONFIG.skipCheck) :
+   !getFromUserSettings('bbb_skip_check_audio', APP_CONFIG.skipCheck);
```

Comment:
In the else part (:), should this not be just "true" instead of !getFromUserSettings('bbb_skip_check_audio', APP_CONFIG.skipCheck);